src/Credentials.js

```diff
@@ -22,6 +22,9 @@ export default class Credentials {
     if (params.requested) {
       payload.requested = params.requested
     }
+    if (params.verified) {
```

Comment:
@zachferland verified will be returned as a list of attestation JWT's from mobile